src/main/java/seedu/address/logic/ModuleInfoSearcher.java

```diff
@@ -11,7 +11,7 @@
  * Logic for searching a module from the Computer Science curriculum.
  */
 public class ModuleInfoSearcher {
-    public static final String MESSAGE_FAILED_TO_FIND_MODULE = "Failed to find module";
+    public static final String MESSAGE_FAILED_TO_FIND_MODULE = "Failed to find module: %s";
     public static final String MESSAGE_EMPTY_SEARCH = "Search is empty";
```

Comment:
Should we put these in the Messages class?

Reply:
Yeah I was thinking of this. I'll go change it